src/rrdp.rs

```diff
@@ -38,36 +47,95 @@ use crate::xml::decode::{Content, Error as XmlError, Reader, Name};
 /// This type represents the decoded content of the RRDP Update Notification
 /// File. It can be read from a reader via the [`parse`][Self::parse]
 /// function. All elements are accessible as attributes.
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub struct NotificationFile {
     /// The identifier of the current session of the server.
     ///
     /// Delta updates can only be used if the session ID of the last processed
     /// update matches this value.
-    pub session_id: Uuid,
-
+    session_id: Uuid,
+    
```

Comment:
Nit: Might want to remove those unnecessary spaces.

Reply:
done